### ISSUE TITLE
Azure Service Bus - Support

### ISSUE THREAD

**Issue by jonatansantana** [bschmitt/laravel-amqp#101]
Hello, this package have azure service bus support or any prediction of implementing it using a proxy proxy or the amqp protocol itself?

**Comment by bschmitt** [comment]
This package just provides support for AMQP, maybe I'm mistaken but Azure also supports that? But in general, any other protocols are not planned.

**Comment by jonatansantana** [comment]
Yes, azure servoce bus provide support for AMQP, but, 1.0 version.
Can use?
